### ISSUE TITLE
JSON type is broken for array. Array comes back as `string`.

### ISSUE THREAD

**Issue by thesunny** [prisma/prisma#2619]
## Bug description

When you do a `prisma.table.create` or `prisma.table.update` with an array value, the array is converted to a serialized `string`.

## How to reproduce

Based on a minimal schema on PostgreSQL:

```
model stories {
  id    Int    @id @default(autoincrement())
  title String
  tags  Json
}
```

```ts
import { PrismaClient } from "@prisma/client"

const prisma = new PrismaClient()

async function start() {
  const data = { title: "Cool", tags: ["Matrix", "Star Wars"] }
  const createStory = await prisma.stories.create({ data })
  console.log(createStory)
  console.log(typeof createStory.tags) // returns `string`
  const story = (await prisma.stories.findMany({ take: 1 }))[0]
  console.log(story)
  console.log(typeof story.tags) // returns `string`
}

start()
```

Array data in a JSON field is always serialized (i.e. turned into a `string`). For clarity, I have checked the database and in the database, the value is also a `string`.

![image](https://user-images.githubusercontent.com/469587/83379721-10765f80-a391-11ea-8c33-404e87eb6fbf.png)

`string`, `number` and `object` work okay.

## Expected behavior

When you insert an `array` it should be returned as an `array`

## Prisma information

Shared above

## Environment & setup

- OS: Mac OS
- Database: PostgreSQL
- Prisma version: Prisma 2, beta 7
- Node.js version: 12.16.2

**Comment by thesunny** [comment]
A similar issue was marked as fixed in https://github.com/prisma/prisma/issues/2432 but I am on beta 7 which is where the fix is supposed to be; however, this is a different issue.

Prisma does not throw an error, but instead saves the data as a `string` instead of `array`.

**Comment by pantharshit00** [comment]
I can confirm the behavior and I do believe in this is a bug

**Comment by Jolg42** [comment]
This is now fixed in 2.0.0-alpha.1286

Thanks for the 👌 issue!
